core/unit_test/CMakeLists.txt

```diff
@@ -546,9 +536,17 @@ IF(KOKKOS_ENABLE_OPENACC AND KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewAPI_f.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewMapping_b.cpp
     ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_ViewResize.cpp
+    ${CMAKE_CURRENT_BINARY_DIR}/openacc/TestOpenACC_BitManipulationBuiltins.cpp
     )
 endif()
 
+# FIXME_NVHPC: Old NVHPC compilers (V22.5 or older) fail to compile
+IF(KOKKOS_ENABLE_SERIAL AND KOKKOS_CXX_COMPILER_ID STREQUAL NVHPC)
+  list(REMOVE_ITEM Serial_SOURCES2
+    ${CMAKE_CURRENT_BINARY_DIR}/serial/TestSerial_TeamBasic.cpp
+    )
+ENDIF()
```

Comment:
We don't support older NVHPC compiler.  We can upgrade the SDK in the CI build and require a more recent version.

Reply:
Then, if we choose NVHPC V22.7 as the minimum version, we can re-enable the disabled unit tests.
(In fact, I couldn't check whether V22.6 works or not because I don't have an access  to NVHPC V22.6)

Reply:
Restored unit tests that were disabled for old NVHPC compilers (V22.5 or older).
The issues on NVHPC V22.5 are related to supporting atomic operations on float/double variables, which are necessary features. 
Therefore, it will be better to change the minimum required version of NVHPC to V22.7 for the OpenACC backend.